core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -243,6 +249,49 @@ public static int ascii(String s) {
         ? 0 : s.codePointAt(0);
   }
 
+  /** SQL REPEAT(string, int) function. */
+  public static String repeat(String s, int n) {
+    if (n < 1) {
+      return "";
+    }
+    int len = s.length() * n;
+    StringBuilder buf = new StringBuilder(len);
+    for (int i = 0; i < n; i++) {
+      buf.append(s);
+    }
+    return buf.toString();
```

Comment:
from L257~262, you can use Strings.repeat

Reply:
Will do.